src/pages/api/revalidate-paths.ts

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import type { NextApiRequest, NextApiResponse } from 'next/types'
+import { StatusCodes } from 'http-status-codes'
+import { validateToken } from 'lib/api-validate-token'
+
+const PATHS_LIMIT = 100
```

Comment:
This is an arbitrary value for now, right?

Reply:
Yes! Definitely open for input here, felt like an okay limit to start.

Reply:
128? 🤓 (kidding)

Reply:
NERD

Reply:
it is done 🧙🏻